doc/man3/OSSL_PARAM_dup.pod

```diff
@@ -0,0 +1,57 @@
+=pod
+
+=head1 NAME
+
+OSSL_PARAM_dup, OSSL_PARAM_merge, OSSL_PARAM_free
+- OSSL_PARAM copy functions
+
+=head1 SYNOPSIS
+
+ #include <openssl/params.h>
+
+ OSSL_PARAM *OSSL_PARAM_dup(const OSSL_PARAM *p);
+ OSSL_PARAM *OSSL_PARAM_merge(const OSSL_PARAM *p, const OSSL_PARAM *p1);
+ void OSSL_PARAM_free(OSSL_PARAM *p);
+
+=head1 DESCRIPTION
+
+Algorithm parameters can be exported/imported from/to providers using arrays of
+B<OSSL_PARAM>. The following utility functions allow the parameters to be
+duplicated and merged with other B<OSSL_PARAM> to assist in this process.
+
+OSSL_PARAM_dup() duplicates the parameter array I<p>. This function does a deep
+copy of the data. 
+
+OSSL_PARAM_merge() merges the parameter arrays I<p> and I<p1> into a new
+parameter array. If I<p> and I<p1> contain values with the same 'key' then the
+value from I<p1> will replace the I<p> value. This function does a shallow copy
+of the parameters. If I<p1> is NULL then a copy of I<p> is returned.
+
+OSSL_PARAM_free() frees the parameter array I<p> that was created using
+OSSL_PARAM_dup() or OSSL_PARAM_merge().
```

Comment:
Include `OSSL_PARAM_BLD_to_param()` here and mention it in the param builder page.  There is zero reason to have the param builder free function now.

Reply:
Ok